Resources/Prototypes/_NF/Entities/Objects/Shields/shields.yml

```
@@ -8,4 +8,17 @@
   - type: Item
     sprite: _NF/Objects/Weapons/Melee/e_shield.rsi
   - type: PointLight
-    color: green
\ No newline at end of file
+    color: green
+
+- type: entity
+  name: metal shield
+  parent: [RiotShield, BaseC1Contraband ] # Frontier: BaseSecurityContraband<BaseC1Contraband
```

```suggestion
  parent: [BaseC1Contraband, RiotShield ]
```
other order, contraband should take precedent